src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenOverridingAndHiding.vb

```diff
@@ -553,7 +559,88 @@ BC30345: 'Public Shared Sub f([x As Integer = 0])' and 'Public Shared Sub f(ByRe
                ~
 </errors>)
         End Sub
+        <Fact()>
+        Public Sub OverloadingBasedOnOptionalParameters_B()
+            Dim useOpts = _ImplicitDefaultOptionalParameter_
+            If Not InternalSyntax.Parser.CheckFeatureAvailability(useOpts, InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter) Then
```

Comment:
Note: Used to check to see if this feature is enabled, and thus run the new unittests. This is due to potentially different errors being produced ( if any ).